ci-operator/jobs/openshift/ci-tools/openshift-ci-tools-master-presubmits.yaml

```diff
@@ -148,6 +148,61 @@ presubmits:
         secret:
           secretName: regcred
     trigger: (?m)^/test( | .* )images,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    cluster: build01
+    context: ci/prow/images-debug
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-ci-tools-master-images-debug
+    rerun_command: /test images-debug
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --give-pr-author-access-to-namespace=true
+        - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
+        - --kubeconfig=/etc/apici/kubeconfig
+        - --target=[images]
+        command:
+        - ci-operator
```

Comment:
This only ensures the ci-operator pod is running on the debug node.
However, we need the real workload (the builds) to run there.

We need to unwrap the ci-op process.

Reply:
I am thinking get `builds.yaml` and add tollerant and node-selector there.
and use the rehearsal namespace (remove the `ttl`) and trigger the builds by `oc apply -f builds.yaml`

Reply:
```
oc --context build01 explain build.spec.nodeSelector
KIND:     Build
VERSION:  build.openshift.io/v1

FIELD:    nodeSelector <map[string]string>

DESCRIPTION:
     nodeSelector is a selector which must be true for the build pod to fit on a
     node If nil, it can be overridden by default build nodeselector values for
     the cluster. If set to an empty map or a map with any values, default build
     nodeselector values are ignored.

oc --context build01 explain build.spec.tolerations
error: field "tolerations" does not exist
```

Reply:
Asked via slack: https://coreos.slack.com/archives/CE2HALN2W/p1589232462315800